crontabber/tests/test_crontabber.py

```diff
@@ -2149,6 +2150,105 @@ def test_job_list_separated_sloppily(self):
             ok_('Ran FooJob' in infos, infos)
             ok_('Ran BarJob' in infos, infos)
 
+    def test_run_two_jobs_almost_simultaneously_first_time(self):
```

Comment:
In order for this to really test if crontabber is trying to run them at them at the same time,  the transactions need to be guaranteed to overlap each other.  I suggest patching the transaction executor for both. It should hold one transaction open (before the update) until the other completes its select.